package.json

```diff
@@ -38,21 +32,27 @@
     "@typescript-eslint/parser": "^5.15.0",
     "ajv": "^6.12.6",
     "ajv-cli": "^5.0.0",
+    "css-loader": "^6.9.1",
+    "dotenv": "^16.4.1",
     "eslint": "^8.11.0",
     "eslint-config-prettier": "^9.0.0",
     "eslint-plugin-prettier": "^5.0.0",
     "eslint-plugin-react-hooks": "^4.6.0",
     "eslint-plugin-storybook": "^0.6.13",
-    "react": "^18.2.0",
-    "react-dom": "^18.2.0",
+    "prettier": "^3.0.1",
```

Comment:
did you forgot to add a config?

Reply:
no, it's there, just moved some dependencies to devDeps